Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -3269,7 +3270,7 @@ void WebPageProxy::processNextQueuedMouseEvent()
         m_process->startResponsivenessTimer();
     }
 
-    std::optional<Vector<SandboxExtension::Handle>> sandboxExtensions;
+    std::optional<Vector<SandboxExtension::Handle>> sandboxExtensions = std::nullopt;
```

Comment:
= std::nullopt is unnecessary and redundant with the default std::optional constructor.